### PR TITLE
mute_process should not always return true

### PR DESCRIPTION
**Pull request by wwang71119**

### CODE REVIEW

**Comment by wwang71119** [comment]
Already tested and it always return true.

File: src/lib.rs [hunk 1]
```diff
@@ -741,16 +741,13 @@ impl EsClient {
...
-            es_mute_process(client.client, token);
-        }
+        let client = (*self.client).lock();
+        let client = match client {
+            Ok(c) => c,
+            Err(_) => return false,
+        };
 
-        true
+        0 == unsafe { es_mute_process(client.client, token) }
```
Comment (jvimal-eg) [inline src/lib.rs]: Can you use es_return_t_ES_RETURN_SUCCESS constant defined already?

**Review by jvimal-eg — APPROVED** [review]
lgtm, assuming this compiles fine.  We don't have a github action set up for this repo.

**Comment by wwang71119** [comment]
it compiles

**Pull request merged**